odoo/addons/test_new_api/tests/test_properties.py

```diff
@@ -734,6 +734,54 @@ def test_properties_field_integer_float_boolean(self):
             self.message_1.attributes[2]['value'], False,
             msg='Boolean value must have been converted to False')
 
+        # When the user sets the value 0 for the property fields of type integer
+        # and float, the system should store the value 0 and shouldn't transform
+        # 0 to False (-> unset value).
+
+        self.message_1.attributes = {'int_value': 0, 'float_value': 0}
+        self.assertEqual(len(self.message_1.attributes), 3)
```

Comment:
note that we can use _get_sql_properties to even check the value stored in database